### DISCUSSION TITLE
404: NOT_FOUND

### DISCUSSION THREAD

**Discussion by iamshaaani** [vercel/community#5985] [Help]
### Summary

My NextJS Project url: https://iamshaaanip.vercel.app working fine. 
I redirect project url to: https://iamshaaani.co but it show 
404: NOT_FOUND
Code: DEPLOYMENT_NOT_FOUND
ID: bom1::tc5l4-1709301265918-c692a187a014

![error](https://github.com/vercel/community/assets/54396807/a29458a2-082a-458c-aeca-7753e38aa54c)

### Steps to Reproduce

Please guide, How can I solve this problem

**Comment by reachtrevor** [comment]
Having the same issue even when using the starter kit blog example here. https://vercel.com/templates/next.js/blog-starter-kit

**Reply by iamshaaani** [reply]
Thanks it, due to DNS issue, I solved it myself.

**Reply by reachtrevor** [reply]
Interesting, mine is not DNS related. The generated vercel domains are not even working

**Comment by reachtrevor** [comment]
Okay, I had noticed in the served assets area that it only had static assets like images etc. Turns out the project did not auto detect the framework as Nextjs so when it tried to build it missed building the actual application.

Just changed the framework to Nextjs and it worked again 👍🏼
